### DISCUSSION TITLE
ATI Rage emulation and Mach 8

### DISCUSSION THREAD

**Discussion by ghost** [86Box/86Box#4490] [Ideas]
Mach 8 and Rage 3D to Rage 128 Pro emulation

**Comment by TC1995** [comment]
Mach8 emulation is already there.

**Comment by Torinde** [comment]
[QEMU](https://www.kraxel.org/blog/2019/09/display-devices-in-qemu/) "ati vga" has model=rage128p selects the "Rage 128 Pro" and model=rv100 selects the "Radeon RV100" (DX7). "Most important 2D accel ops are implemented too. 3D acceleration is not implemented yet."; https://github.com/joncampbell123/dosbox-x/issues/3889
